### DISCUSSION TITLE
Font Weight Inconsistency: React Native Component Looks Different on Android and iOS

### DISCUSSION THREAD

**Discussion by premdasvm** [nativewind/nativewind#605] [Q&A]
Hi, 

I'm currently facing an issue with a React Native component that contains a Text element. When I apply the `font-semibold` style, I noticed that the font weight change is taking effect in iOS, but there's no change in Android. As a result, my component looks different on the two platforms

```jsx
<View className="p-6 m-3 bg-white border border-gray-200 rounded-3xl shadow">
	<View className="gap-1">
		<View className="flex-row justify-between">
			<View>
				<Text className="text-2xl font-semibold tracking-tight text-gray-900 ">
					Good Morning,
				</Text>
				<Text className="text-2xl font-bold tracking-tight text-gray-900 ">
					Premdas
				</Text>
			</View>
			<View>
				<Image
					source={{
						uri: "https://img.freepik.com/premium-photo/cartoonish-3d-animation-boy-glasses-with-blue-hoodie-orange-shirt_899449-25777.jpg",
					}}
					className="flex items-center justify-center w-16 h-16 mx-2 overflow-hidden rounded-full"
				/>
			</View>
		</View>
		<Text className="text-sm font-light tracking-tight text-gray-900 ">
			You're not Check-in yet Today.
		</Text>
		<Text className="font-bold tracking-tight text-blue-500">10:59 AM</Text>
		<CheckInButton />
	</View>
</View>
```

I've attached screenshots for reference. The first screenshot is from my Android device, and the second one is from iOS.

I'd appreciate any insights or suggestions on how to ensure consistent font weight across both Android and iOS platforms. Is there a workaround or something I might be missing?

<img width="359" alt="Screenshot 2023-11-03 at 12 26 26 AM" src="https://github.com/marklawlor/nativewind/assets/14838550/d37df4d2-c200-4e40-91c9-d080bc16e02f">

<img width="344" alt="Screenshot 2023-11-03 at 12 26 43 AM" src="https://github.com/marklawlor/nativewind/assets/14838550/771a722b-39fb-41e6-9d38-86b52495fa18">

**Comment by marklawlor** [comment]
https://reactnative.dev/docs/text-style-props#fontweight

Can you first test using inline styles that these variants are available for your font family.

**Reply by premdasvm** [reply]
Native variants wasn't working either, then i added some custom fonts with different variants, that seems to do the trick.
